docs/content/advanced/backend/taxes/inclusive-pricing.md

```diff
@@ -0,0 +1,188 @@
+# Tax Inclusive Pricing
+
+In this document, you’ll learn how tax-inclusive pricing works in Medusa.
+
+:::note
+
+Tax Inclusive Pricing is currently in beta mode and guarded by a feature flag. To enable it, set the environment variable `MEDUSA_FF_TAX_INCLUSIVE_PRICING` to `true`, and [run migrations](../migrations.md#migrate-command).
+
+:::
+
+## Introduction
+
+Countries have different tax rates, even if they share the same currency. In these cases, using one tax-exclusive price for a currency that is used by multiple countries or regions can show different price to the customer based on the tax rate of each region.
```

Comment:
suggestion: before going into this let's cover what tax-inclusive pricing means. E.g.:

> Tax Inclusive pricing allows you to set the final prices for products and shipping options regardless of the customer's applicable tax rates. When tax-inclusive prices are used, Medusa will automatically calculate the tax amount for a given price.

> This can be useful when two countries use the same currency but have different tax rates. If you want your prices to be the same across these countries, you will have to manage two price lists to account for the tax differences. Using tax-inclusive pricing you only have to specify the price once.